nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/main/java/org/apache/nifi/snmp/processors/SNMPWorker.java

```diff
@@ -47,7 +46,7 @@
      * @param snmp instance of {@link Snmp}
      * @param target instance of {@link AbstractTarget}
      */
-    public SNMPWorker(Snmp snmp, AbstractTarget target) {
+    protected SNMPWorker(Snmp snmp, AbstractTarget target) {
```

Comment:
In this specific case the `public` modifier didn't make any sense at all and can't have been used outside of subclasses because it is in an abstract class.

For all other cases I've already commented separately.